fw/http.c

```diff
@@ -2992,6 +2992,18 @@ tfw_h1_adjust_req(TfwHttpReq *req)
 	int r;
```

Comment:
Just a note to keep in mind with the upcoming changes. This function is called in a callback when we're about to forward the request to an upstream, i.e. when it can not be satisfied by the cache. Meantime `tfw_cache_purge_method()` is called without the callback (see `tfw_cache_do_action()`) and do not return a status code. It seems you'll need to update the workflow a little bit.